### PR TITLE
Add `url` as configurable parameter

### PR DESCRIPTION
**Pull request by Amir-P**
## :scroll: Description
Added url as configurable parameter

## :bulb: Motivation and Context
To use this plugin with on-premise Sentry servers

## :green_heart: How did you test it?
With our own on-premise Sentry server!

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] I updated the docs if needed
- [ ] All tests passing
- [x] No breaking changes

### CODE REVIEW

**Comment by marandaneto** [comment]
@Amir-P thanks for doing that.

**Comment by marandaneto** [comment]
@Amir-P please fix the formatting and add a changelog, so we can merge and release, thanks

**Comment by Amir-P** [comment]
@marandaneto Done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Unreleased
+# 1.0.0-alpha.5
+
+* Feat: Ability to configure url for on-premise server (#17)
```
Comment (marandaneto) [inline CHANGELOG.md:3]:
```suggestion
# Unreleased

* Feat: Ability to configure url for on-premise server (#17)
```

Comment (marandaneto) [inline CHANGELOG.md:3]:
```suggestion
# Unreleased

* Feat: Ability to configure url for on-premise server (#17)
```

Comment (marandaneto) [reply CHANGELOG.md:3]: since we change the versioning during releasing

Comment (Amir-P) [reply CHANGELOG.md]: Right. Done.

**Comment by marandaneto** [comment]
@brustolin or @philipphofmann  would you like to have a look?

**Review by brustolin — APPROVED** [review]
LGTM

**Review by marandaneto — APPROVED** [review]

**Pull request merged**